src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequences.scss

```diff
@@ -18,6 +18,14 @@
   background-color: $white;
 }
 
+.smallScreenHeader {
+  top: 49px;
+}
```

Comment:
What's this for?

https://user-images.githubusercontent.com/6834224/167392653-f0bdb01f-2f68-42b4-acb7-d28b5a7cbd90.mp4

`top` wouldn't work on `position: static` (which is the default for a DOM element)